docker/docker-compose.quickstart.yaml

```diff
@@ -5,17 +5,26 @@ services:
     build:
       context: ../
       dockerfile: docker/images/Dockerfile.quickstart
+      args:
+        - NPM_TOKEN
     image: edb/docs-dev-server
     container_name: docs-dev-server
     hostname: docs-dev-server
     env_file:
       - ../.env.development
     init: true
-    working_dir: /app
-    command: >
-      sh -c "npm run install-dependencies &&
-      npm run develop -- -H 0.0.0.0 -p ${PORT:-8000} 2>&1"
+    command: npm run develop -- -H 0.0.0.0 -p ${PORT:-8000} 2>&1
```

Comment:
It seems to be unhappy with the `2>&1` being here. I get an error when trying to start the container:
```
docs-dev-server    | Unknown argument: 2>&1
```

Otherwise, this looks good to me

Reply:
Fixed.